### ISSUE TITLE
Support for PHP 8.2

### ISSUE THREAD

**Issue by Gelbotron** [laravel-shift/workbench-issues#91]
**What would you like to do?**
We have new projects in PHP 8.2, would appreciated if we could use Workbench with them.

**Before/After code snippets**
`>>> Event: Processing paths: app, database/factories, database/seeders, config, database/migrations, routes, tests
Fatal error: Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.2.0". You are running 8.1.17. in /project/vendor/composer/platform_check.php on line 24
Failed to run rector: convert-switch-to-match: #0 `

**Comment by RomkaLTU** [comment]
Yes that is very strange just tried to run but no PHP 8.2 support.
